neqo-transport/src/crypto.rs

```diff
@@ -1010,6 +1011,7 @@ impl CryptoStates {
                     "[{self}] Continue packet numbers for initial after retry (write is {:?})",
                     prev.rx.used_pn,
                 );
+                initial.rx.continuation(&prev.rx)?;
```

Comment:
This one is similar to the situation below, except that this code is only really used in a context that might matter at a server.  The situation with servers is far less problematic because servers are the first to commit to a version and clients can only commit to a version after the server has done so.  This is still necessary, but I'm completely confident in this path.